worlds/dkc3/__init__.py

```diff
@@ -118,7 +121,13 @@ def generate_basic(self):
 
         total_junk_count = total_required_locations - len(itempool)
 
-        itempool += [self.create_item(ItemName.bear_coin)] * total_junk_count
+        junk_pool = []
+        junk_keys = list(junk_table.keys())
```

Comment:
https://docs.python.org/3/library/random.html#random.choices (random.choices) should allow you to convert this loop into a single call, example exists in Subnautica.